<!DOCTYPE html>
<html>
	<head>
		<title>R-GenomeInfoDb-1.28.0-2.fc35 - Fedora Packages</title>
		<link rel="stylesheet" href="../../assets/css/bootstrap.min.css">
		<link rel="stylesheet" href="../../assets/css/main.css">
		<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<meta charset="UTF-8">
	</head>
	<body>
		<div class="container">
			<div class="row">
				<a href="/" class="col-md-6 d-flex align-items-center justify-content-center justify-content-md-start p-1">
					<img src="../../assets/images/logo.png">
				</a>
				<div class="col-md-6 d-flex align-items-center">
					
				</div>
			</div>

			<hr />

			<h1>
				R-GenomeInfoDb-1.28.0-2.fc35<small class="text-muted"> in Fedora 35</small>
			</h1>
			<p>
				<a href=".">&crarr; Retrun to the main page of R-GenomeInfoDb</a><br>
				<a href="https://koji.fedoraproject.org/koji/search?match=exact&type=build&terms=R-GenomeInfoDb-1.28.0-2.fc35">View in Koji</a><br>
				<a href="https://bodhi.fedoraproject.org/updates/?search=R-GenomeInfoDb-1.28.0-2.fc35">Search Bodhi</a>
			</p>

			<p>
				
				<b>Package Info</b><br>
				
				<a href="#changelog">ðŸ — Changelog</a><br>
				
				<a href="#dependencies">ðŸ — Dependencies</a><br>
				
				
				<a href="#provides">ðŸ — Provides</a><br>
				
				
				<a href="#files">ðŸ — Files</a><br>
				
			</p>

			<h2 id="changelog">Changelog</h2>
			<div class="table-responsive">
				<table class="table table-striped table-borderless ">
					<thead>
						<tr>
							<th scope="col">Date</th>
							<th scope="col">Author</th>
							<th scope="col">Change</th>
						</tr>
					</thead>
					
					<tr scope="row">
						<td>2021-07-21</td>
						<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.28.0-2</td>
						<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_35_Mass_Rebuild</td>
					</tr>
					
					<tr scope="row">
						<td>2021-06-16</td>
						<td>Tom Callaway <spot at fedoraproject dot org> - 1.28.0-1</td>
						<td>- update to 1.28.0
- Rebuilt for R 4.1.0</td>
					</tr>
					
					<tr scope="row">
						<td>2021-02-03</td>
						<td>Tom Callaway <spot at fedoraproject dot org> - 1.26.2-1</td>
						<td>- update to 1.26.2</td>
					</tr>
					
					<tr scope="row">
						<td>2021-01-25</td>
						<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.24.0-3</td>
						<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
					</tr>
					
					<tr scope="row">
						<td>2020-07-27</td>
						<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.24.0-2</td>
						<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
					</tr>
					
					<tr scope="row">
						<td>2020-06-08</td>
						<td>Tom Callaway <spot at fedoraproject dot org> - 1.24.0-1</td>
						<td>- update to 1.24.0
- rebuild for R 4</td>
					</tr>
					
					<tr scope="row">
						<td>2020-01-28</td>
						<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.22.0-3</td>
						<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
					</tr>
					
					<tr scope="row">
						<td>2019-11-21</td>
						<td>Elliott Sales de Andrade <quantum dot analyst at gmail dot com> - 1.22.0-2</td>
						<td>- Exclude Suggests for unavailable packages</td>
					</tr>
					
					<tr scope="row">
						<td>2019-11-06</td>
						<td>Tom Callaway <spot at fedoraproject dot org> - 1.22.0-1</td>
						<td>- update to 1.22.0</td>
					</tr>
					
					<tr scope="row">
						<td>2019-08-12</td>
						<td>Elliott Sales de Andrade <quantum dot analyst at gmail dot com> - 1.16.0-5</td>
						<td>- Remove explicit dependencies provided by automatic dependency generator</td>
					</tr>
					
					<tbody>
					</tbody>
				</table>
			</div>

			<div class="row">
				
				<div class="col">
					<h2 id="dependencies">Dependencies</h2>
					<ul>
						
						<li>
						
						<a href="/pkgs/R-BiocGenerics/fedora-35.html">
						
						
						R-BiocGenerics</a> >= 0.37.0
						

						</li>
						
						<li>
						
						<a href="/pkgs/R-GenomeInfoDbData/fedora-35.html">
						
						
						R-GenomeInfoDbData
						

						</li>
						
						<li>
						
						<a href="/pkgs/R-IRanges/fedora-35.html">
						
						
						R-IRanges</a> >= 2.13.12
						

						</li>
						
						<li>
						
						<a href="/pkgs/R-RCurl/fedora-35.html">
						
						
						R-RCurl
						

						</li>
						
						<li>
						
						<a href="/pkgs/R-S4Vectors/fedora-35.html">
						
						
						R-S4Vectors</a> >= 0.25.12
						

						</li>
						
						<li>
						
						<a href="/pkgs/R-core/fedora-35.html">
						
						
						R-core</a> = 4.1
						

						</li>
						
						<li>
						
						<a href="/pkgs/texlive-collection-latexrecommended/fedora-35.html">
						
						
						texlive-collection-latexrecommended
						

						</li>
						
					</ul>
				</div>
				
				
				<div class="col">
					<h2 id="provides">Provides</h2>
					<ul>
						
						<li>R(GenomeInfoDb)</li>
						
						<li>R-GenomeInfoDb</li>
						
					</ul>
				</div>
				
			</div>

			
			<h2 id="files">Files</h2>

			<div class="tree">
				<ul>
				
				
				<li>usr/<ul>
				
				
				
				<li>share/<ul>
				
				
				
				<li>R/<ul>
				
				
				
				<li>library/<ul>
				
				
				
				<li>GenomeInfoDb/<ul>
				
				
				
				<li>DESCRIPTION</li>
				
				
				
				<li>INDEX</li>
				
				
				
				<li>Meta/<ul>
				
				
				
				<li>Rd.rds</li>
				
				
				
				<li>features.rds</li>
				
				
				
				<li>hsearch.rds</li>
				
				
				
				<li>links.rds</li>
				
				
				
				<li>nsInfo.rds</li>
				
				
				
				<li>package.rds</li>
				
				
				
				<li>vignette.rds</li>
				
				
				
				</ul></li>
				
				
				
				<li>NAMESPACE</li>
				
				
				
				<li>NEWS</li>
				
				
				
				<li>R/<ul>
				
				
				
				<li>GenomeInfoDb</li>
				
				
				
				<li>GenomeInfoDb.rdb</li>
				
				
				
				<li>GenomeInfoDb.rdx</li>
				
				
				
				</ul></li>
				
				
				
				<li>doc/<ul>
				
				
				
				<li>Accept-organism-for-GenomeInfoDb.R</li>
				
				
				
				<li>Accept-organism-for-GenomeInfoDb.Rnw</li>
				
				
				
				<li>Accept-organism-for-GenomeInfoDb.pdf</li>
				
				
				
				<li>GenomeInfoDb.R</li>
				
				
				
				<li>GenomeInfoDb.Rnw</li>
				
				
				
				<li>GenomeInfoDb.pdf</li>
				
				
				
				<li>index.html</li>
				
				
				
				</ul></li>
				
				
				
				<li>extdata/<ul>
				
				
				
				<li>assembly_accessions.rda</li>
				
				
				
				<li>dataFiles/<ul>
				
				
				
				<li>Arabidopsis_thaliana.txt</li>
				
				
				
				<li>Caenorhabditis_elegans.txt</li>
				
				
				
				<li>Canis_familiaris.txt</li>
				
				
				
				<li>Cyanidioschyzon_merolae.txt</li>
				
				
				
				<li>Drosophila_melanogaster.txt</li>
				
				
				
				<li>Homo_sapiens.txt</li>
				
				
				
				<li>Mus_musculus.txt</li>
				
				
				
				<li>Oryza_sativa.txt</li>
				
				
				
				<li>Populus_trichocarpa.txt</li>
				
				
				
				<li>README</li>
				
				
				
				<li>Rattus_norvegicus.txt</li>
				
				
				
				<li>Saccharomyces_cerevisiae.txt</li>
				
				
				
				<li>Zea_mays.txt</li>
				
				
				
				<li>genomeMappingTbl.csv</li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				<li>help/<ul>
				
				
				
				<li>AnIndex</li>
				
				
				
				<li>DEFAULT_CIRC_SEQS.html</li>
				
				
				
				<li>GenomeDescription-class.html</li>
				
				
				
				<li>GenomeDescription.html</li>
				
				
				
				<li>GenomeInfoDb-internals.html</li>
				
				
				
				<li>GenomeInfoDb.rdb</li>
				
				
				
				<li>GenomeInfoDb.rdx</li>
				
				
				
				<li>Seqinfo-class.html</li>
				
				
				
				<li>Seqinfo.html</li>
				
				
				
				<li>[,Seqinfo-method.html</li>
				
				
				
				<li>aliases.rds</li>
				
				
				
				<li>as.data.frame,Seqinfo-method.html</li>
				
				
				
				<li>as.data.frame.Seqinfo.html</li>
				
				
				
				<li>available.species.html</li>
				
				
				
				<li>bsgenomeName,GenomeDescription-method.html</li>
				
				
				
				<li>bsgenomeName.html</li>
				
				
				
				<li>checkCompatibleSeqinfo.html</li>
				
				
				
				<li>class:GenomeDescription.html</li>
				
				
				
				<li>class:Seqinfo.html</li>
				
				
				
				<li>coerce,DataFrame,Seqinfo-method.html</li>
				
				
				
				<li>coerce,data.frame,Seqinfo-method.html</li>
				
				
				
				<li>commonName,GenomeDescription-method.html</li>
				
				
				
				<li>commonName.html</li>
				
				
				
				<li>dropSeqlevels.html</li>
				
				
				
				<li>extractSeqlevels.html</li>
				
				
				
				<li>extractSeqlevelsByGroup.html</li>
				
				
				
				<li>fetchExtendedChromInfoFromUCSC.html</li>
				
				
				
				<li>genome,ANY-method.html</li>
				
				
				
				<li>genome,Seqinfo-method.html</li>
				
				
				
				<li>genome.html</li>
				
				
				
				<li>genome<-,ANY-method.html</li>
				
				
				
				<li>genome<-,Seqinfo-method.html</li>
				
				
				
				<li>genome<-.html</li>
				
				
				
				<li>genomeBuilds.html</li>
				
				
				
				<li>genomeStyles.html</li>
				
				
				
				<li>getChromInfoFromEnsembl.html</li>
				
				
				
				<li>getChromInfoFromNCBI.html</li>
				
				
				
				<li>getChromInfoFromUCSC.html</li>
				
				
				
				<li>intersect,Seqinfo,Seqinfo-method.html</li>
				
				
				
				<li>isCircular,ANY-method.html</li>
				
				
				
				<li>isCircular,Seqinfo-method.html</li>
				
				
				
				<li>isCircular.html</li>
				
				
				
				<li>isCircular<-,ANY-method.html</li>
				
				
				
				<li>isCircular<-,Seqinfo-method.html</li>
				
				
				
				<li>isCircular<-.html</li>
				
				
				
				<li>keepSeqlevels.html</li>
				
				
				
				<li>keepStandardChromosomes.html</li>
				
				
				
				<li>length,Seqinfo-method.html</li>
				
				
				
				<li>listOrganisms.html</li>
				
				
				
				<li>loadTaxonomyDb.html</li>
				
				
				
				<li>mapGenomeBuilds.html</li>
				
				
				
				<li>mapSeqlevels.html</li>
				
				
				
				<li>merge,NULL,Seqinfo-method.html</li>
				
				
				
				<li>merge,Seqinfo,NULL-method.html</li>
				
				
				
				<li>merge,Seqinfo,Seqinfo-method.html</li>
				
				
				
				<li>merge,Seqinfo,missing-method.html</li>
				
				
				
				<li>merge,missing,Seqinfo-method.html</li>
				
				
				
				<li>names,Seqinfo-method.html</li>
				
				
				
				<li>names<-,Seqinfo-method.html</li>
				
				
				
				<li>orderSeqlevels.html</li>
				
				
				
				<li>organism,GenomeDescription-method.html</li>
				
				
				
				<li>organism.html</li>
				
				
				
				<li>paths.rds</li>
				
				
				
				<li>provider,GenomeDescription-method.html</li>
				
				
				
				<li>provider.html</li>
				
				
				
				<li>providerVersion,GenomeDescription-method.html</li>
				
				
				
				<li>providerVersion.html</li>
				
				
				
				<li>rankSeqlevels.html</li>
				
				
				
				<li>registered_NCBI_assemblies.html</li>
				
				
				
				<li>registered_UCSC_genomes.html</li>
				
				
				
				<li>releaseDate,GenomeDescription-method.html</li>
				
				
				
				<li>releaseDate.html</li>
				
				
				
				<li>releaseName,GenomeDescription-method.html</li>
				
				
				
				<li>releaseName.html</li>
				
				
				
				<li>renameSeqlevels.html</li>
				
				
				
				<li>restoreSeqlevels.html</li>
				
				
				
				<li>seqinfo,GenomeDescription-method.html</li>
				
				
				
				<li>seqinfo.html</li>
				
				
				
				<li>seqinfo<-.html</li>
				
				
				
				<li>seqlengths,ANY-method.html</li>
				
				
				
				<li>seqlengths,Seqinfo-method.html</li>
				
				
				
				<li>seqlengths.html</li>
				
				
				
				<li>seqlengths<-,ANY-method.html</li>
				
				
				
				<li>seqlengths<-,Seqinfo-method.html</li>
				
				
				
				<li>seqlengths<-.html</li>
				
				
				
				<li>seqlevels,ANY-method.html</li>
				
				
				
				<li>seqlevels,Seqinfo-method.html</li>
				
				
				
				<li>seqlevels-wrappers.html</li>
				
				
				
				<li>seqlevels.html</li>
				
				
				
				<li>seqlevels0.html</li>
				
				
				
				<li>seqlevels<-,ANY-method.html</li>
				
				
				
				<li>seqlevels<-,Seqinfo-method.html</li>
				
				
				
				<li>seqlevels<-.html</li>
				
				
				
				<li>seqlevelsInGroup.html</li>
				
				
				
				<li>seqlevelsInUse,CompressedList-method.html</li>
				
				
				
				<li>seqlevelsInUse,Vector-method.html</li>
				
				
				
				<li>seqlevelsInUse.html</li>
				
				
				
				<li>seqlevelsStyle,ANY-method.html</li>
				
				
				
				<li>seqlevelsStyle,Seqinfo-method.html</li>
				
				
				
				<li>seqlevelsStyle,character-method.html</li>
				
				
				
				<li>seqlevelsStyle.html</li>
				
				
				
				<li>seqlevelsStyle<-,ANY-method.html</li>
				
				
				
				<li>seqlevelsStyle<-,Seqinfo-method.html</li>
				
				
				
				<li>seqlevelsStyle<-,character-method.html</li>
				
				
				
				<li>seqlevelsStyle<-.html</li>
				
				
				
				<li>seqnames,GenomeDescription-method.html</li>
				
				
				
				<li>seqnames,Seqinfo-method.html</li>
				
				
				
				<li>seqnames.html</li>
				
				
				
				<li>seqnames<-,Seqinfo-method.html</li>
				
				
				
				<li>seqnames<-.html</li>
				
				
				
				<li>show,GenomeDescription-method.html</li>
				
				
				
				<li>show,Seqinfo-method.html</li>
				
				
				
				<li>sortSeqlevels,ANY-method.html</li>
				
				
				
				<li>sortSeqlevels,character-method.html</li>
				
				
				
				<li>sortSeqlevels.html</li>
				
				
				
				<li>species,GenomeDescription-method.html</li>
				
				
				
				<li>species.html</li>
				
				
				
				<li>standardChromosomes.html</li>
				
				
				
				<li>summary,Seqinfo-method.html</li>
				
				
				
				<li>summary.Seqinfo.html</li>
				
				
				
				</ul></li>
				
				
				
				<li>html/<ul>
				
				
				
				<li>00Index.html</li>
				
				
				
				<li>GenomeDescription-class.html</li>
				
				
				
				<li>GenomeInfoDb-internals.html</li>
				
				
				
				<li>R.css</li>
				
				
				
				<li>Seqinfo-class.html</li>
				
				
				
				<li>fetchExtendedChromInfoFromUCSC.html</li>
				
				
				
				<li>getChromInfoFromEnsembl.html</li>
				
				
				
				<li>getChromInfoFromNCBI.html</li>
				
				
				
				<li>getChromInfoFromUCSC.html</li>
				
				
				
				<li>loadTaxonomyDb.html</li>
				
				
				
				<li>mapGenomeBuilds.html</li>
				
				
				
				<li>rankSeqlevels.html</li>
				
				
				
				<li>seqinfo.html</li>
				
				
				
				<li>seqlevels-wrappers.html</li>
				
				
				
				<li>seqlevelsStyle.html</li>
				
				
				
				</ul></li>
				
				
				
				<li>registered/<ul>
				
				
				
				<li>NCBI_assemblies/<ul>
				
				
				
				<li>Apis_mellifera.R</li>
				
				
				
				<li>Arabidopsis_thaliana.R</li>
				
				
				
				<li>Betacoronavirus.R</li>
				
				
				
				<li>Bos_taurus.R</li>
				
				
				
				<li>Brassica_napus.R</li>
				
				
				
				<li>Caenorhabditis_elegans.R</li>
				
				
				
				<li>Callithrix_jacchus.R</li>
				
				
				
				<li>Camponotus_floridanus.R</li>
				
				
				
				<li>Canis_lupus_familiaris.R</li>
				
				
				
				<li>Chlamydomonas_reinhardtii.R</li>
				
				
				
				<li>Danio_rerio.R</li>
				
				
				
				<li>Drosophila_melanogaster.R</li>
				
				
				
				<li>Drosophila_virilis.R</li>
				
				
				
				<li>Galleria_mellonella.R</li>
				
				
				
				<li>Gallus_gallus.R</li>
				
				
				
				<li>Gasterosteus_aculeatus.R</li>
				
				
				
				<li>Harpegnathos_saltator.R</li>
				
				
				
				<li>Homo_sapiens.R</li>
				
				
				
				<li>Hordeum_vulgare.R</li>
				
				
				
				<li>Macaca_fascicularis.R</li>
				
				
				
				<li>Macaca_mulatta.R</li>
				
				
				
				<li>Madurella_mycetomatis.R</li>
				
				
				
				<li>Microtus_ochrogaster.R</li>
				
				
				
				<li>Monodelphis_domestica.R</li>
				
				
				
				<li>Mus_musculus.R</li>
				
				
				
				<li>Mustela_putorius_furo.R</li>
				
				
				
				<li>Pan_paniscus.R</li>
				
				
				
				<li>Pan_troglodytes.R</li>
				
				
				
				<li>Patiria_miniata.R</li>
				
				
				
				<li>Rattus_norvegicus.R</li>
				
				
				
				<li>Saccharomyces_cerevisiae.R</li>
				
				
				
				<li>Sus_scrofa.R</li>
				
				
				
				<li>Taeniopygia_guttata.R</li>
				
				
				
				<li>Vitis_vinifera.R</li>
				
				
				
				</ul></li>
				
				
				
				<li>UCSC_genomes/<ul>
				
				
				
				<li>README.TXT</li>
				
				
				
				<li>apiMel1.R</li>
				
				
				
				<li>apiMel2.R</li>
				
				
				
				<li>bosTau1.R</li>
				
				
				
				<li>bosTau2.R</li>
				
				
				
				<li>bosTau3.R</li>
				
				
				
				<li>bosTau4.R</li>
				
				
				
				<li>bosTau6.R</li>
				
				
				
				<li>bosTau7.R</li>
				
				
				
				<li>bosTau8.R</li>
				
				
				
				<li>bosTau9.R</li>
				
				
				
				<li>calJac3.R</li>
				
				
				
				<li>canFam1.R</li>
				
				
				
				<li>canFam2.R</li>
				
				
				
				<li>canFam3.R</li>
				
				
				
				<li>canFam4.R</li>
				
				
				
				<li>ce10.R</li>
				
				
				
				<li>ce11.R</li>
				
				
				
				<li>ce2.R</li>
				
				
				
				<li>ce4.R</li>
				
				
				
				<li>ce6.R</li>
				
				
				
				<li>danRer10.R</li>
				
				
				
				<li>danRer11.R</li>
				
				
				
				<li>danRer5.R</li>
				
				
				
				<li>danRer6.R</li>
				
				
				
				<li>danRer7.R</li>
				
				
				
				<li>dm1.R</li>
				
				
				
				<li>dm2.R</li>
				
				
				
				<li>dm3.R</li>
				
				
				
				<li>dm6.R</li>
				
				
				
				<li>eboVir3.R</li>
				
				
				
				<li>galGal2.R</li>
				
				
				
				<li>galGal3.R</li>
				
				
				
				<li>galGal4.R</li>
				
				
				
				<li>galGal5.R</li>
				
				
				
				<li>galGal6.R</li>
				
				
				
				<li>gasAcu1.R</li>
				
				
				
				<li>hg15.R</li>
				
				
				
				<li>hg16.R</li>
				
				
				
				<li>hg17.R</li>
				
				
				
				<li>hg18.R</li>
				
				
				
				<li>hg19.R</li>
				
				
				
				<li>hg38.R</li>
				
				
				
				<li>macFas5.R</li>
				
				
				
				<li>mm10.R</li>
				
				
				
				<li>mm39.R</li>
				
				
				
				<li>mm8.R</li>
				
				
				
				<li>mm9.R</li>
				
				
				
				<li>monDom5.R</li>
				
				
				
				<li>musFur1.R</li>
				
				
				
				<li>panPan1.R</li>
				
				
				
				<li>panPan2.R</li>
				
				
				
				<li>panPan3.R</li>
				
				
				
				<li>panTro1.R</li>
				
				
				
				<li>panTro2.R</li>
				
				
				
				<li>panTro3.R</li>
				
				
				
				<li>panTro4.R</li>
				
				
				
				<li>panTro5.R</li>
				
				
				
				<li>panTro6.R</li>
				
				
				
				<li>rheMac10.R</li>
				
				
				
				<li>rheMac2.R</li>
				
				
				
				<li>rheMac3.R</li>
				
				
				
				<li>rheMac8.R</li>
				
				
				
				<li>rn1.R</li>
				
				
				
				<li>rn2.R</li>
				
				
				
				<li>rn3.R</li>
				
				
				
				<li>rn4.R</li>
				
				
				
				<li>rn5.R</li>
				
				
				
				<li>rn6.R</li>
				
				
				
				<li>sacCer1.R</li>
				
				
				
				<li>sacCer2.R</li>
				
				
				
				<li>sacCer3.R</li>
				
				
				
				<li>susScr11.R</li>
				
				
				
				<li>susScr2.R</li>
				
				
				
				<li>susScr3.R</li>
				
				
				
				<li>taeGut1.R</li>
				
				
				
				<li>taeGut2.R</li>
				
				
				
				<li>wuhCor1.R</li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				<li>unitTests/<ul>
				
				
				
				<li>test_Seqinfo-class.R</li>
				
				
				
				<li>test_mapGenomeBuilds.R</li>
				
				
				
				<li>test_rankSeqlevels.R</li>
				
				
				
				<li>test_seqlevels-wrappers.R</li>
				
				
				
				<li>test_seqlevelsStyle.R</li>
				
				
				
				<li>test_utils.R</li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
				
				</ul></li>
				
				
			</div>
			
		</div>
	</body>
</html>

